### PR TITLE
file => 5.43

### PR DESCRIPTION
**Pull request by saltedcoffii**
## Description
Upgrade `file` and `libmagic` to version 5.43.

## Additional information

Works properly (all tested in containers):
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

#### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/saltedcoffii/chromebrew.git CREW_TESTING_BRANCH=file_5.43 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/filecmd.rb [hunk 1]
```diff
@@ -1,74 +1,83 @@
...
-    i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/filecmd/5.40-2_i686/filecmd-5.40-2-chromeos-i686.tpxz',
-    x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/filecmd/5.40-2_x86_64/filecmd-5.40-2-chromeos-x86_64.tpxz'
-  })
-  binary_sha256({
-    aarch64: '651c9ebb6b6294969df3f776b49fdb3eb08371a7ab8eb929a8e2c8d298e9d6bd',
-    armv7l: '651c9ebb6b6294969df3f776b49fdb3eb08371a7ab8eb929a8e2c8d298e9d6bd',
-    i686: 'ea5a96dbdf900f836b1ca69eefa72b2622722a1f3d0e46736932cc75b12a653f',
-    x86_64: '4d0da2febbb0f5a9e5d08dfe2026a9816aa2b1fb603d2c9328082e740a96c8a2'
-  })
+  source_sha256 '8c8015e91ae0e8d0321d94c78239892ef9dbc70c4ade0008c0e95894abfb1991'
```
Comment (uberhacker) [inline packages/filecmd.rb:12]: Do you think you can add binaries with @satmandu's scripts?

File: packages/filecmd.rb [hunk 2]
```diff
@@ -1,74 +1,83 @@
...
+
     @filefix = <<~EOF
-      #!/bin/bash
-      for f in $(find . -name configure); do
-        sed -i 's,/usr/bin/file,#{CREW_PREFIX}/bin/file,g' ${f}
-      done
+      #!/usr/bin/env bash
+      while IFS= read -r -d '' f; do
+        sed -i 's,/usr/bin/file,#{CREW_PREFIX}/bin/file,g' "${f}"
+      done <  <(find . -name configure -print0)
```
Comment (uberhacker) [inline packages/filecmd.rb:30]: No need to rewrite but whatever. :)

Comment (saltedcoffii) [reply packages/filecmd.rb:30]: https://www.shellcheck.net/wiki/SC2044

File: packages/filecmd.rb (hunk 1 above)
Comment (saltedcoffii) [reply packages/filecmd.rb:12]: I'm having trouble with them as I don't have all the scripts

File: packages/filecmd.rb (hunk 2 above)
Comment (uberhacker) [reply packages/filecmd.rb:30]: If we ever use spaces or brackets in filenames then we have a bigger problem but this is good to know.

File: packages/filecmd.rb (hunk 1 above)
Comment (uberhacker) [reply packages/filecmd.rb:12]: What are you missing?  @satmandu provided all the scripts in `#general`.

Comment (saltedcoffii) [reply packages/filecmd.rb:12]: - gl.sh api key
- crewb
- crewbk or whatever letter we're at now
- a functional armv7l container

Comment (saltedcoffii) [reply packages/filecmd.rb:12]: binaries are built and uploaded @uberhacker (a5faa5f)

**Review by satmandu — APPROVED** [review]

**Pull request merged**